ompi/mca/pml/ob1/pml_ob1_recvreq.c

```diff
@@ -428,6 +430,30 @@ static int mca_pml_ob1_recv_request_put_frag (mca_pml_ob1_rdma_frag_t *frag)
 
     /* send rdma request to peer */
     rc = mca_bml_base_send (bml_btl, ctl, MCA_PML_OB1_HDR_TYPE_PUT);
+
+#ifdef SOFTWARE_EVENTS_ENABLE
+    volatile int64_t bytes_sent;
+    unsigned int i;
+    if(attached_event[OMPI_BYTES_SENT_USER] == 1){
+        if(recvreq->req_recv.req_base.req_tag >= 0){
+            bytes_sent = 0;
+            for(i = 0; i < ctl->des_segment_count; i++){
+                bytes_sent += ctl->des_segments[i].seg_len;
+            }
+            SW_EVENT_RECORD(OMPI_BYTES_SENT_USER, bytes_sent);
```

Comment:
This was my mistake.  I will change this to be similar to the get recording code in the mca_pml_ob1_recv_request_get_frag function.